docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.8'
+
+services:
+  nemascan:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    entrypoint: nemascan-nxf.sh
+    environment:
+      - VCF_VERSION=20220216
+      - WORK_DIR=gs://caendr-nf-pipelines/workdir/rbv218
```

Comment:
hum....we might want to remove these values here, replace them with placeholders, and add some info in the `README.md` about how to populate them.

Reply:
For now, I've just removed the docker compose file entirely: c3b4a8252bfd84a4d05528f7040557522d949771